### DISCUSSION TITLE
Ban Appeal

### DISCUSSION THREAD

**Discussion by davidstown** [Rapptz/discord.py#9893] [General] [answered]
Hello there, I would like to appeal my ban of the discord.py Discord Server. I heard that I was banned for "racism", but when I got banned, I didn't have access to my account, which means it was compromised. Thanks.

My Discord username is damimaa, my ID is 1001076524612603945

**Comment by AbstractUmbra** [comment, marked as answer]
I've reviewed your account and it was indeed banned in 2022 for racism. I'll remove that ban for now but any follow-up offences will be a permanent ban.

In future, find a moderator of the server and DM them, not use the library GitHub for ban appeals.

**Reply by davidstown** [reply]
Thank you so much!
